apps/docs/content/guides/database/replication/etl.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: ETL
+description: Replicate your Supabase database to external destinations
+---
+
+`pg_replicate` simplifies the process of syncing your data to an external destination. This could be another database, a data warehouse, or a specialized service such as customer management or business analytics software.
```

Comment:
```suggestion
`ETL` simplifies the process of syncing your data to an external destination. This could be another database, a data warehouse, or a specialized service such as customer management or business analytics software.
```